File: servico-musica-java/java.rest/src/main/java/servico/java/rest/controller/MusicaController.java
package servico.java.rest.controller;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;
import servico.java.rest.entity.Musica;
import servico.java.rest.repository.MusicaRepository;

import java.util.List;

@RestController
@RequestMapping("/servico-rest-java/musica")
public class MusicaController {

    @Autowired
    private MusicaRepository musicaRepository;

    @GetMapping("/todas-musicas")
    public List<Musica> todasAsMusicas(){
        return musicaRepository.findAll();
    }
}

File: servico-musica-kotlin/kotlin.rest/src/main/kotlin/servico/kotlin/rest/controller/MusicaController.kt
package servico.kotlin.rest.controller

import org.springframework.beans.factory.annotation.Autowired
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController
import servico.kotlin.rest.entity.Musica
import servico.kotlin.rest.repository.MusicaRepository

@RestController
@RequestMapping("/servico-rest-kotlin/musica")
class MusicaController {
    @Autowired
    private val musicaRepository: MusicaRepository? = null

    @GetMapping("/todas-musicas")
    fun todasAsMusicas(): List<Musica> {
        return musicaRepository!!.findAll()
    }
}